src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -358,34 +358,37 @@ public function makePathRelative($endPath, $startPath)
             $startPath = str_replace('\\', '/', $startPath);
         }
 
+        $stripDriveLetter = function ($path) {
+            if (strlen($path) > 2 && substr($path, 1, 2) === ':/' && ctype_alpha($path[0])) {
```

Comment:
I used `substr()` because it’s done that way in `isAbsolutePath()` [Filesystem.php:587](https://github.com/symfony/symfony/blob/7b8409ad373d7d1613ce313047272f495c80bda8/src/Symfony/Component/Filesystem/Filesystem.php#L587). Should I change it though?